fortytwo.cabal

```diff
@@ -1,5 +1,5 @@
 name:                fortytwo
-version:1.0.6
+version:             1.1.0
```

Comment:
please avoid bumping the project version. That's something I will eventually do by myselves